data-prepper-expression/src/test/java/org/opensearch/dataprepper/expression/ConditionalExpressionEvaluatorIT.java

```diff
@@ -129,6 +129,9 @@ private static Stream<Arguments> validExpressionArguments() {
                 Arguments.of("true == (/is_cool == true)", event("{\"is_cool\": true}"), true),
                 Arguments.of("not /is_cool", event("{\"is_cool\": true}"), false),
                 Arguments.of("/status_code < 300", event("{\"status_code\": 200}"), true),
+                Arguments.of("/status_code != null", event("{\"status_code\": 200}"), true),
+                Arguments.of("/response == null", event("{\"status_code\": 200}"), true),
+                Arguments.of("/response != null", event("{\"status_code\": 200}"), false),
```

Comment:
I recommend adding a few test cases with null on the left had side to test operators do not throw unexpected exceptions.

An example where a bug could occur:
Given a hypothetical operator `foo` that checks for class equality implemented as `(lhs, rhs) -> lhs.getClass().equals(rhs.getClass())`

When invoked `null foo null`

Then a `NullPointerException` will be thrown instead of `true`

Reply:
I can add that. I see no harm in adding more tests.

Reply:
Added few test cases with null on the left side.

Reply:
Thanks!